sdk/eventhub/event-hubs/src/linkEntity.ts

```diff
@@ -162,9 +183,9 @@ export class LinkEntity {
    * @returns {void}
    */
   protected async _ensureTokenRenewal(): Promise<void> {
-    const tokenValidTimeInSeconds = this._context.tokenProvider.tokenValidTimeInSeconds;
-    const tokenRenewalMarginInSeconds = this._context.tokenProvider.tokenRenewalMarginInSeconds;
-    const nextRenewalTimeout = (tokenValidTimeInSeconds - tokenRenewalMarginInSeconds) * 1000;
+    if (!this._tokenTimeout) {
+      throw new Error("_tokenTimeout cannot be null");
```

Comment:
In what scenario will this if block be entered?
Even if it is, what can the user possibly do with this error?
We should either use a default or just simply exit